hack/create-repos.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")
+source "${SCRIPT_ROOT}/repos.sh"
+
+if [ "$#" = 0 ] || [ "$#" -gt 2 ]; then
+    echo "usage: $0 [source-github-user-name] dest-github-user-name"
+    echo
+    echo "This connects to git@github.com:<from>/<repo>. Set GITHUB_HOST to access git@<GITHUB_HOST>:<from>/<repo> instead."
+    exit 1
+fi
+
+FROM="kubernetes"
+TO="${1}"
+if [ "$#" -ge 2 ]; then
+    FROM="${TO}"
+    TO="${2}"
+fi
+GITHUB_HOST=${GITHUB_HOST:-github.com}
+
+repo_count=${#repos[@]}
+
+# safety check
+if [ "${TO}" = "kubernetes" ]; then
```

Comment:
Maybe an env var for forbidden destinations?

Reply:
this snippet is from the other shell script, let's update both at the same time after this PR lands please.